support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx

```diff
@@ -90,11 +90,11 @@ const content = (
     >
       <CampaignHeader />
       <Content>
-        <Text title="Open up your world view, Weekly">
-          <LargeParagraph>Inside the essential magazine from
-          The&nbsp;Guardian, you&rsquo;ll find expert opinion, insight and culture, curated to
-          bring you a progressive, international perspective. You&rsquo;ll also discover
-          challenging new puzzles every week. Subscribe today and get delivery worldwide.
+        <Text title="Catch up on the issues that matter">
+          <LargeParagraph>The Guardian Weekly magazine is a round-up of the world news,&#32;
+            opinion and long reads that have shaped the week. Inside, the past seven days&apos;&#32;
+            most memorable stories are reframed with striking photography and insightful companion&#32;
+            pieces, all handpicked from the Guardian and the Observer.
```

Comment:
Capitals on `The Guardian and The Observer` too

Reply:
Really? That's so awkward 😬 

Not doubting you, but my inner editor is horrified.